### DISCUSSION TITLE
Subsectors and delta in lap layout

### DISCUSSION THREAD

**Discussion by Jathu05** [theOehrly/Fast-F1#134] [Q&A]
Can we get a lap layout which shows the delta between 2 drivers? Just like the speed layout but for lap delta and also can we get a layout which shows the fastest in a sector or sub sector?? ![image](https://user-images.githubusercontent.com/101971998/160255523-6fa3786e-0b44-4d2a-8143-bfc54b96657f.jpeg)

**Comment by theOehrly** [comment]
Well, the code for the delta time exists. You'd just need to adapt the plotting a bit, so that you can plot those values. That said, the delta time calculation is not very accurate, so the results won't necessarily be good.

For mini sector comparisons, you can find a pretty good script here: https://github.com/theOllieS/F1_Minisectors (not mine)
